### DISCUSSION TITLE
Going to to file from links?

### DISCUSSION THREAD

**Discussion by corpooo** [nvim-neorg/neorg#125] [Q&A] [answered]
Let's say I have the following structure

```sh
notes/
    index.norg
    test.norg
```

What should I add in `index.norg` to have a link to `test.norg` when I press `<CR>`?

I have tried the following and they don't seem to work:
`[test](test.norg)`
`[test](/test.norg)`
`[test](./test.norg)`

Also, after I run `Neorg` from a non `.norg` file with no arguments the first time it works - it takes me to my setup workspace. But after I run `Neorg return` and I go back, when I run `Neorg`,  I get
```
E471: Argument required
```
Does anyone have an idea why?

**Comment by vhyrro** [comment, marked as answer]
Hey man! As of right now Neorg doesn't have a syntax to link to a file *specifically*. Currently our syntax only allows for linking to an element *within* another file, and the syntax looks like so:
```
[test](:other_file:#An element in that file)
```
You cannot do:
```
[test](:other_file:)
```
Maybe if there's a serious use for it we could support it? We'll see as time goes on.

Regarding your issue with the `Neorg` command, this has now been fixed in the `unstable` branch. There was a bit of confusion because `:Neorg` actually had a double meaning - when not loaded fully, `:Neorg` meant "kickstart the neorg environment", whilst when Neorg was loaded `:Neorg` actually meant "execute a command for Neorg". Now when Neorg isn't fully loaded the command is called `:NeorgStart` to avoid that sort of confusion.

`:Neorg return` currently only returns from Neorg files to your old workspace. I'll probably change `:Neorg return` to `:Neorg switch` and give it more functionality like being able to toggle between Neorg and your code with just one command :)

**Reply by kkharji** [reply]
Oh so liking to another file (no specific element) is not yet supported?

**Reply by vhyrro** [reply]
Nope not yet. Quite a lot of people are asking about it so I'm considering it - it feels like our existing syntax isn't quite built for it though, we'll just hafta wait and see i guess :)

**Reply by kkharji** [reply]
hmm at first I thought `[test](:other_file:*)` might work, or `[test](:other_file)` anyways it would be cool if this is supported. Thanks

**Reply by mrossinek** [reply]
We've had some discussion on discord and think that `[test](:other_file:)` will probably match the rest the closest. We might also drop support for including multiple files (i.e. `[test](:file:other_file:#target)` will no longer be valid) [frankly we don't see a good reason to keep it...]

**Reply by 777uliahshafar** [reply]
May I asked what syntax highlight corespond to :other_file:
